gift-box-shop-frontend/src/components/ShopMenu.js

```diff
@@ -0,0 +1,11 @@
+import styles from '../../styles/components/ShopMenu.module.scss';
+
+export default function ShopMenu({stepsList}) {
+    return (
+        <div className={styles.containerShopMenu}>
+            {stepsList.map(step=>(
+                <p className={step.check? styles.selectedOption:''} key={step.title.split('.')[0]}>{step.title}</p> 
```

Comment:
This is a clickable element, so we should rather use a Button instead of a Paragraph and style it accordingly.